doc/select.md

```diff
@@ -54,6 +99,10 @@ crud.select('developers', {{'>=', 'age', 30}})
 ...
 ```
 
+**Note**: index is named ``age_index``, but we can still query by the field name ``age``
+and the search will be done using index without a full scan. If the names of index and
+field match, the search will also be performed using index.
```

Comment:
It would be more intuitive if we would have an example of using an index name in a condition. Now it is not obvious that it is possible to scan over an index not only using a field name.

Reply:
Fixed